core/jazz_ui/src/app/secondary-components/create-service/oss/create-service.component.ts

```diff
@@ -406,7 +543,11 @@ export class CreateServiceComponent implements OnInit {
   //function to validate event source names
   validateEvents(value){
     if(value != null && ((value[0] === '-' || value[value.length - 1] === '-') || (value[0] === '.' || value[value.length - 1] === '.') || (value[0] === '_' || value[value.length - 1] === '_'))){
-      this.invalidEventName = true;
+      if(this.typeOfPlatform === 'aws') {
+      this.invalidEventName = true; }
```

Comment:
Nit: Watch bracketing of blocks. Don't worry about it now. Pending addition of lint to project.

Reply:
ok